packages/tools/create-package/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@kadena-dev/create-package",
+  "version": "0.0.0",
+  "description": "Create package from template",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/kadena-community/kadena.js.git",
+    "directory": "packages/tools/create-package"
+  },
+  "license": "ISC",
+  "type": "module",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "bin": {
+    "create-package": "lib/index.js"
+  },
+  "files": [
+    "dist",
+    "lib"
+  ],
+  "scripts": {
+    "build": "tsc",
+    "dev": "tsc --watch",
+    "format": "npm run format:md && npm run format:pkg && npm run format:src",
+    "format:ci": "prettier config src --check",
+    "format:md": "remark README.md -o --use @kadena-dev/markdown",
+    "format:pkg": "sort-package-json",
+    "format:src": "prettier src --write",
+    "lint": "eslint ./src --ext .js,.ts --fix",
+    "test": ""
+  },
+  "dependencies": {
+    "ink": "^4.1.0",
+    "ink-text-input": "~5.0.1",
+    "react": "^18.2.0"
```

Comment:
> I'm not a fan of depending on React when building tools. This should have minimal dependencies, and definitely not intrusive ones that can break the whole monorepo, like react.

How could it break the whole monorepo? I mean, in the end it's just another dependency we're using "everywhere". That's more a flaw of using monorepos (with strict versioning) in general than of this particular package.

> We want this tool usable outside of this repo, so react should be at max a `peerDependency`.

I think peer dependencies are not useful for stand-alone tools. It should be possible to run it directly using e.g. `npm init ...` etc.

However, we could (should?) bundle the whole thing. Then we can move such dependencies to `devDependencies`.

> I'd rather see that we avoid this by using commander and inquirer? Those are also much more popular in the community: https://npmtrends.com/commander-vs-ink-vs-inquirer

Let's make the same comparison for monorepo tools ;-) Jokes aside, today's popularity isn't a great driver for innovation. Looking at https://github.com/vadimdemedes/ink#whos-using-ink I think more and more people are starting to see how great this model is. E.g. Cloudflare's new wrangler CLI tool has been praised all around. And when Sir npm himself uses it in [node-tap](https://github.com/tapjs/node-tap) I know we're not doing it wrong.

Reply:
> That's more a flaw of using monorepos

It's probably a flaw of a combination of react requiring only one version to be available and npm not resolving to a specific version somehow, nonetheless, it's an issue we have to deal with. 

Furthermore, using ink encourages an unstructured way of doing a simple task: asking questions. When reviewing the code a few things catch my eye. Things that would otherwise be abstracted away. Sure, they can still be abstracted away, but that becomes our code, not the library code, and thus more maintainable code:

* I see in the JSX a list of options, that would otherwise be abstracted away into a list.
* It's not clear from the code which questions depend on answers from other questions

> And when Sir npm himself uses it in [node-tap](https://github.com/tapjs/node-tap) I know we're not doing it wrong 

I understand why TAP is using it, he's basically building a full-blown application in the CLI, that requires state management, async actions and events to be handled, all while doing tasks in the background. 

This package is asking literally 5 questions, in an unstructured way, **while** introducing complexity adding a whole state management for just 3 states: 'cancelled', 'done', 'default' which is only introduced _because_ of using state like that.

Reply:
Like I said in our chat, this is totally over-engineered for this specific one. But I think it's a great way to build CLIs in general. We already have the `@kadena/kda-cli` tool and to me it makes sense to build on that and share the knowledge to ultimately set high standards for our DX/tooling.

Reply:
> We already have the @kadena/kda-cli tool built with Ink

Even for `@kadena/kda-cli` it's not a done deal that we will move forward with `ink`. In its functionality, it is similar to this one; a one-time bootstrapper. 

As I said, when a CLI becomes an application, as in it "requires state management, async actions and events to be handled, all while doing tasks in the background", then it's nice to have a more powerful tool in how to deal with that, for this, not so much.

> Like I said in our chat, this is totally over-engineered for this specific one

And I don't see this cli getting the requirements to justify `ink` at this point